### ISSUE TITLE
Bug when fetching BPLvl from a private account

### ISSUE THREAD

**Issue by Luc1412** [Terbau/fortnitepy#86]
When I try to use `client.fetch_battlepass_level()` from an account which is set to Private I get an internal error instead of `Forbidden`

Error:
```py
raceback (most recent call last):
  File "/red32/lib/python3.8/site-packages/redbot/core/dev_commands.py", line 192, in _eval
    result = await func()
  File "<string>", line 4, in func
  File "/bot-configs/bot2/cogs/Downloader/lib/fortnitepy/client.py", line 2151, in fetch_battlepass_level
    return data[user_id]
KeyError: '369644c6224d4845aa2b00e63b60241d'
 
```

**Comment by Terbau** [comment]
Thanks for reporting. A fix will be published with the next release :)

**Comment by Luc1412** [comment]
This also breaks with fetch_multiple. It didn't include the user in the dict.

**Comment by Terbau** [comment]
This has been fixed in 2.0. Thanks for reporting :)
